lab02-kaylee/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "lab02-kaylee",
+  "version": "1.0.0",
+  "description": "lab02",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "lint": "eslint ./lib",
```

Comment:
Good job remembering your lint script. No points lost for this since I don't think Scott covered it in detail, but you're missing one more dependency of ``eslint`` order to make this run properly, but no worries.